examples/tools/crosshairs.md

```diff
@@ -1,6 +1,38 @@
 ---
-layout: needs-example
+layout: tool-simple
 title: Crosshairs Tool
 toolName: Crosshairs
 toolType: stack
 ---
+
+
+<h2 class="title is-2">How to set up the {{page.toolName}} tool:</h2>
+
+{% highlight javascript %}
+// Init cornerstone tools
+cornerstoneTools.init()
+
+// Enable any elements, and display images
+const imageIds = [`image_id_1`,`image_id_2`,]
```

Comment:
https://github.com/cornerstonejs/cornerstoneTools/pull/914/files#diff-59fbbac36b28851384a6cae1fbe1eb73R50

```js
const scheme = 'wadouri';
const baseUrl = 'https://mypacs.com/dicoms/';
const series = [
    'image_1.dcm',
    'image_2.dcm',
];

const imageIds = series.map(seriesImage => `${scheme}:${baseUrl}${seriesImage}`);
```

Something like the above snippet may better illustrate:

1. That `scheme`s are used to determine which image loader we use to load the image.
2. A more common scenario for how we would locate the image with a unique URI/ID (a URL that returns the `.dcm` resource).

Reply:
Nice! Thank you for sharing that. Already updated.